### ISSUE TITLE
Diverse Warnings in backend

### ISSUE THREAD

**Issue by katipierson** [timber/debug-bar-timber#21]
Getting Warnings in backend - two times - after update yesterday. Plugin "Debug Bar" is up to date.

WARNUNG: wp-content/plugins/debug-bar-timber/class-debug-bar-timber.php:5 - include(vendor/autoload.php): failed to open stream: No such file or directory

Tab "Timber" is empty.

**Comment by manzwebdesigns** [comment]
For the Timber tab to have anything, you must be on a page that uses it.  Is this a new install or update to an existing install of the plugin?

Thanks for using the plugin and reporting this!

**Comment by jarednova** [comment]
@manzwebdesigns it appears that the `/vendor` folder (with autoload and the other composer dependencies) did not deploy with the GitHub action. Can you take a look at that this week? In the meantime, we should rollback the `readme.txt` filed with WordPress.org to use the prior version while we get this sorted out.

No biggie! after a long period of not giving this plugin much (er, any) attention there's bound to be some hiccups.

**Comment by manzwebdesigns** [comment]
I plan to do that tomorrrow.  I am excited for the opportunity to give back to the community that helps me earn my living!

**Comment by katipierson** [comment]
I've used it on my development system first. In the development system I only updated the plugin. When the error occurred I uninstalled and reinstalled it there.

The mistake remained. So I installed a completely empty Wordpress and then only installed the Timber, Debug Bar and Timber Debug Bar plugins. The error occurred again.

After that I'm pretty sure that it wasn't my system and reported the error here. I hope that was correct.
